src/main/java/org/verdictdb/core/resulthandler/ExecutionResultReader.java

```diff
@@ -63,4 +63,8 @@ public DbmsQueryResult next() {
   public void remove() {
     throw new UnsupportedOperationException();
   }
+
+  public boolean hasEndofQueueReached() {
```

Comment:
How is this different from hasNext() defined above?

Reply:
It seems redundant. I will remove it.